girder/api/sftp.py

```diff
@@ -261,32 +262,25 @@ def shutdown_request(self, request):
         pass
 
 
-def _main():  # pragma: no cover
+@click.command(name='sftpd', short_help='Run the Girder SFTP service.')
+@click.option('-i', '--identity-file', help='path to identity (private key) file')
+@click.option('-p', '--port', default=DEFAULT_PORT, type=int)
+@click.option('-H', '--host', default='localhost')
+def _main(identity_file, port, host):  # pragma: no cover
```

Comment:
Should this be `main`? Should everything else be `_main`? I don't really know, but it ought to be consistent.

Reply:
It doesn't matter to me. I think there's consensus that CLI functions in Python aren't/shouldn't be part of our API surface. Perhaps when all of the entrypoints are moved into a central module (#2612) we can just mark that module as outside of the API surface.